$OpenBSD$
Tests fail due to incorrect _POSIX_TIMERS value in <unistd.h>.
Allow functions to be actually used.
--- qtbase/src/concurrent/qtconcurrentiteratekernel.cpp.ports.orig	Mon Nov  3 22:55:57 2014
+++ qtbase/src/concurrent/qtconcurrentiteratekernel.cpp	Mon Nov  3 22:56:56 2014
@@ -79,7 +79,7 @@ static qint64 getticks()
 
 static qint64 getticks()
 {
-#if defined(_POSIX_TIMERS) && (_POSIX_TIMERS > 0)
+#if defined(Q_OS_OPENBSD) || (defined(_POSIX_TIMERS) && (_POSIX_TIMERS > 0))
     clockid_t clockId;
 
 #ifndef _POSIX_THREAD_CPUTIME
